wordpress-mobile/WordPress-iOS
# Upload progress coordinator

Fixes #8126 

This PR adds hooks to the MediaUploadCoordinator to report progress using the MediaProgressCoordinator.

At the moment this only logs progress on the console when using the new feature flag async upload.

To test:
 - Open the app
 - Select a blog
 - Open the media library
 - Tap on the plus and select the async option
 - Add multiple photos
 - See the progress being reported on the console.
